src/loader/bgw_counter.c

```diff
@@ -55,13 +55,13 @@ extern void
 bgw_counter_setup_gucs(void)
 {
 
-	DefineCustomIntVariable("timescaledb.max_bgw_processes",
+	DefineCustomIntVariable("timescaledb.max_background_workers",
 							"Maximum background worker processes allocated to TimescaleDB",
 							"Max background worker processes allocated to TimescaleDB - set to at least 1 + number of databases in Postgres instance to use background workers ",
 							&guc_max_background_workers,
 							guc_max_background_workers,
 							0,
-							max_worker_processes,
+							1000, /* no reasonable way to have more than 1000 background workers */
```

Comment:
I could see a reason with a very large multi-core computer, but we can always change this later.